pkg/apis/numaplane/v1alpha1/pipelinerollout_types.go

```diff
@@ -72,13 +72,20 @@ type UpgradingPipelineStatus struct {
 	InterStepBufferServiceName string `json:"interStepBufferServiceName,omitempty"`
 
 	// OriginalScaleMinMax stores for each vertex, the original scale min and max values as JSON string
-	OriginalScaleMinMax []VertexScale `json:"originalScaleMinMax,omitempty"`
+	//OriginalScaleMinMax []VertexScale `json:"originalScaleMinMax,omitempty"`
+	OriginalScaleMinMax []VertexScaleDefinition `json:"originalScaleMinMax,omitempty"`
```

Comment:
in the end, breaking the vertices down into Min and Max values made more sense than creating a json string to hold them, since it wouldn't work to patch with the json string anyway